spec/spec_helper.rb

```diff
@@ -35,8 +35,41 @@ def inside_of_folder(folder)
   end
 end
 
+# See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 RSpec.configure do |config|
-  config.expect_with(:rspec) { |c| c.syntax = :should }
   config.include SpecHelpers::InstanceMethods
   config.extend SpecHelpers::ClassMethods
+
+  config.filter_run_when_matching :focus
+  config.example_status_persistence_file_path = "spec/examples.txt"
+  config.warnings = true
+  config.order = :random
+  Kernel.srand config.seed
+
+  config.expect_with :rspec do |expectations|
```

Comment:
More than happy to add the rspec 4 comment and rspec-4 detection test. Indeed, wouldn't mind stripping down the auto-generated comment as well (I only left it in, in case the team saw value in it).

The reason I added these (as opposed to just ignoring them and dealing with any issues when 4 comes out), is because the current test suite doesn't violate any of them. So a theoretical 4.0 upgrade would be trivial at present. So I figured they could be enabled to prevent any future test from sneaking in a violation that _would_ be an upgrade blocker.